AIDungeon_2.ipynb

```diff
@@ -36,49 +38,201 @@
-        "- **[Support AI Dungeon 2](https://www.patreon.com/bePatron?u=19115449) on Patreon to help me to continue improving the game with all the awesome ideas I have for its future!**\n"
+        "- While you wait you can [read adventures others have had](https://aidungeon.io/)\n",
+        "- [Read more](https://pcc.cs.byu.edu/2019/11/21/ai-dungeon-2-creating-infinitely-generated-text-adventures-with-deep-learning-language-models/) about how AI Dungeon 2 is made.\n",
+        "- **[Support AI Dungeon 2](https://www.patreon.com/bePatron?u=19115449) on Patreon to help me to continue improving the game with all the awesome ideas I have for its future!**"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "pyNN-3UDv0L-"
+      },
+      "source": [
+        "# Main Game"
       ]
     },
     {
       "cell_type": "code",
+      "execution_count": 0,
       "metadata": {
-        "id": "FKqlSCrpS9dH",
+        "colab": {},
         "colab_type": "code",
-        "colab": {}
+        "id": "FKqlSCrpS9dH"
       },
+      "outputs": [],
       "source": [
+        "# Install\n",
         "!git clone --depth 1 --branch master https://github.com/AIDungeon/AIDungeon/\n",
         "%cd AIDungeon\n",
+        "!pip install -r requirements.txt > /dev/null"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "fiywfTj--_Pe"
+      },
+      "outputs": [],
+      "source": [
+        "# Download model from torrent:\n",
         "!./install.sh\n",
-        "from IPython.display import clear_output \n",
+        "from IPython.display import clear_output\n",
         "clear_output()\n",
         "print(\"Download Complete!\")"
-      ],
-      "execution_count": 0,
-      "outputs": []
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": 0,
       "metadata": {
-        "id": "YjArwbWh6XwN",
+        "colab": {},
         "colab_type": "code",
-        "colab": {}
+        "id": "YjArwbWh6XwN"
       },
+      "outputs": [],
       "source": [
+        "# Play\n",
         "from IPython.display import Javascript\n",
         "display(Javascript('''google.colab.output.setIframeHeight(0, true, {maxHeight: 5000})'''))\n",
         "!python play.py"
-      ],
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "kIldfwd8wjvT"
+      },
+      "source": [
+        "# Utilities (Persistent Save / Load, OOM Fix)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "zLx1yMu9wwBg"
+      },
+      "outputs": [],
+      "source": [
+        "# RUN THIS FIRST: Mount Google Drive\n",
```

Comment:
Clarify a little?
-> "If you want to be able to save to your Google Drive, run this first"